distribution/docker/src/docker/Dockerfile

```diff
@@ -257,6 +259,24 @@ RUN sed -i -e 's/ES_DISTRIBUTION_TYPE=tar/ES_DISTRIBUTION_TYPE=docker/' bin/elas
     find . -xdev -perm -4000 -exec chmod ug-s {} + && \\
     find . -type f -exec chmod o+r {} +
 
+<% if (docker_base == "cloud") { %>
+# Preinstall common plugins
+COPY repository-s3-${version}.zip repository-gcs-${version}.zip repository-azure-${version}.zip /opt/
+RUN bin/elasticsearch-plugin install --batch \\
```

Comment:
Should we delete these files after install?

Reply:
This happens in the `builder` image, which we throw away.

Reply:
Ah, right.